telemetry/definitions/commonDefinitions.json

```diff
@@ -6201,6 +6206,21 @@
                 }
             ]
         },
+        {
+            "name": "lambda_createServerlessLandProject",
```

Comment:
maybe it's more useful to use the existing `appBuilder_` prefix? https://github.com/Vandita2020/aws-toolkit-common/blob/8e2fc912aad1bc1c349a83ee99b081d1196e85a5/telemetry/definitions/commonDefinitions.json#L2664